x-pack/qa/sql/src/main/resources/agg_nulls.sql-spec

```diff
@@ -0,0 +1,14 @@
+selectGenderWithNullsAndGroupByGender
+SELECT gender, COUNT(*) count FROM test_emp_with_nulls GROUP BY gender ORDER BY gender;
```

Comment:
Should this be a separate file? Maybe should just be part of the aggs file.

Reply:
@nik9000 thank you for reviewing, much appreciated.
I've done it this way with the idea of refactoring this in the near future as a wider effort (part of https://github.com/elastic/elasticsearch/issues/32079). Then the `test_emp_with_nulls` would disappear and the null values will be part of `test_emp` itself and more tests will be added to various test files (functions, `select`s with `group by`, `IS NULL` type of selects and possibly other sections). I considered the small change of allowing null values as part of the aggregations results worthy of being added now relatively quickly (and allow a minor functionality be available), instead of tackling the null values support wider task which will probably take more time. Also, when the wider null-handling task is considered, I can add the documentation covering the `null` group result.
Let me know your thoughts.